src/screens/Auth/EmailLoginScreen.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { StyleSheet, Text, Image, View, TouchableHighlight, SafeAreaView, Button, Alert, Platform, StatusBar, Dimensions,ImageBackground } from 'react-native'
+import Input from '../../components/Input';
+import LoginButton from '../../components/LoginButton';
+import COLORS from '../../constants';
+import { FontAwesome5 } from '@expo/vector-icons';
+import PasswordInput from '../../components/PasswordInput';
+import Data from "../../Data/data.json"
+import { useState } from 'react';
+import { Link } from 'expo-router'
+
+function EmailLoginScreen() {
+    const [email, setEmail] = useState<string>("")
+    const [password, setPassword] = useState<string>("")
+    
+    const handleLogin = () => {
+        const user = Data.users.find((user) => user.Email === email)
+        if (user) {
+            console.log("user exist")
+        } else {
+            console.log("no")
+        }
+    }
+
+    return (
+        <SafeAreaView style={styles.container}>
+        <View style={styles.parent}>
+            <View style={styles.upper}>
+                <ImageBackground source={require("../../../assets/headImage.png")} style={styles.Image}>
+                    <View style={styles.IntroView}>
+                        <Text style={styles.IntroText}>Glad to see you!!</Text>
+                    </View>
+                </ImageBackground>
+            </View>
+            <View style={styles.LowerBig}>
+
+               <View style={styles.ViewInput}>
+                <Input text="Email"     onChangeText={newText => setEmail(newText)}  value={email}    />
+               <PasswordInput text="Password" onChangeText={newText =>setPassword(newText)}  value={password} />
+                    <View style={styles.Forget}>
+                        <Text style={styles.ForgetText}>Forget password? <Text style={styles.retrieve}>Retrieve</Text></Text>
+                    </View>    
+                
+
+            </View>
+                <View style={styles.footer}>
+                    <View style={styles.ButtonView}>
+                         <LoginButton
+                    title="Login"
+                    onPress={handleLogin}
+                
+                    />
+                    </View>
+                    <View style={styles.LowerLastView}>
+                        <Text style={styles.LastViewFirst}>Don't have an account?<Link href="/screens/RegisterScreen"><Text style={styles.retrieve}>Sign Up</Text></Link> </Text>    
+
+                    </View>
+               
+                
+            </View>
+            
+
+
+
+
+
+
```

Comment:
Kindly remove the empty spaces present in your codebase

Reply:
Done